src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import duke.DukeException;
+import duke.commands.Command;
+import duke.parser.Parser;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.storage.Storage;
+
 public class Duke {
```

Comment:
Perhaps group the relevant classes into packages?

Reply:
Agree, would be better to group them and order them by alphabetical order.